eventing/triggers/new-trigger-filters.adoc

```diff
@@ -0,0 +1,45 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="new-trigger-filters"]
+= New trigger filters
+include::_attributes/common-attributes.adoc[]
+:context: new-trigger-filters
+
+toc::[]
+
+:FeatureName: New trigger filters feature
+include::snippets/technology-preview.adoc[]
+
+The new trigger filters feature adds a new `filters` field to triggers that aligns with the filters API field defined in the CloudEvents Subscriptions API. It allows you to specify filter expressions, where each expression evaluates to `true` or `false` for each event. 
+
+The following example shows a trigger using the new filters field:
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: my-service-trigger
+spec:
+  broker: default
+  filters:
+    - cesql: "source LIKE '%commerce%' AND type IN ('order.created', 'order.updated', 'order.canceled')"
+  subscriber:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: my-service
+----
+
+The `filters` field contains an array of filter expressions, each evaluating to either `true` or `false`. If any expression evaluates to `false`, the event will not be sent to the subscriber. Each filter expression uses a specific dialect that determines the type of filter and the set of allowed additional properties within the expression.
```

Comment:
not sure if that should break down the above `cesql` query - and break it down for explaining?

Reply:
I think we can keep this explanation to the generic `filters` field and not specific to the cesql. WDYT?